src/docfx/lib/json/JsonUtility.cs

```diff
@@ -191,6 +191,17 @@ public static (List<Error> errors, JToken value) Parse(string json, string file)
             }
         }
 
+        public static void Merge(JObject container, params JObject[] overwrites)
+        {
+            if (overwrites == null)
+                return;
+
+            foreach (var overwrite in overwrites)
+            {
+                Merge(container, overwrite);
+            }
+        }
+
         public static void Merge(JObject container, JObject overwrite)
```

Comment:
should this method be `private` and only expose `Merge(JObject container, params JObject[] overwrites)`?